docs/UserGuide.md

```diff
@@ -406,6 +371,34 @@ If unindicated, the `School Residence` field will default to `DOES NOT LIVE ON C
 * No appointment should clash with any other appointments.
 * The student that the appointment is for must exist in the records.
 
+--------------------------------------------------------------------------------------------------------------------
+
+## FAQ
+
+**Q**: What do I do if I accidentally delete a student? 
+<br>
+**A**: Unfortunately, the deletion is permanent. You will have to manually re-add the student.
+
+**Q**: What can be included in medical details?
+<br>
+**A**: Allergies and past medical history.
+
+**Q** : Will I be able to add other types of appointments besides vaccination appointments?
+<br>
+**A** : No, the current version only allows you to add vaccination appointments and not any other type of appointments. 
+
+**Q** : Am I able to reassign an appointment to another student?
+<br>
+**A** : No, you will need to add a new appointment for the new student. 
```

Comment:
Should the ans say "No, you will need to delete the appointment and add a new one for the new student" instead?

Reply:
I think the current version is okay?

Reply:
okie!